### ISSUE TITLE
Unable to interrupt code

### ISSUE THREAD

**Issue by WiliTest** [microsoft/vscode-jupyter#12111]
It's the first time I'm using jupyter (to run [this code](https://pastebin.com/CVSws1mc)) and I'm surprise to don't find the basic button "STOP", to the stop code. I can't even interrupt the code since the button "INTERRUPT" is grayed out.  The only way I found is to restart VScode (closing the tab don't even stop it).

It's a shame because jupyter looks better than the terminal, but without these basic features, I can't use it.

**Comment by DonJayamanne** [comment]
I'm sorry you are running into this issue and thanks for filling this bug
Play could you provide the details we request in the issue template that you have failed to provide, eg version and logs
 
thanks for providing the sample code, I'll try that at my end

**Comment by WiliTest** [comment]
Sorry for the delay. I'm working on it as the issue reporter doesn't seem to work (cf. https://github.com/microsoft/vscode/issues/167415 ). 

Meanwhile I can provide theses information (but I'll be back when the reporter will work).
Visual Studio Code
Version: 1.73.1 (user setup)
Commit: 6261075646f055b99068d3688932416f2346dd3b
Date: 2022-11-09T04:27:29.066Z (2 wks ago)
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19042
Sandboxed: No

**Comment by WiliTest** [comment]
To make it short: I could not reproduce the bug (that's why I'm closing this issue). It's now working as expected (the stop button is not anymore grayed out) . I reinstalled all my python versions, so it could have been caused by this. 

Jupiter wasn't able to break a while loop (I apologize: it's a duplicate). The code I'm running (cf. the link above) opens an external window that contains a picture taken from the webcam (each time you press the keyboard key "p" it adds an image to a GIF, it creates and saves the gif when pressing "q"). I did not see the windows open, so the code was waiting for me to press "p" or "q". I was not able to stop as the stop button was grayed out.
